pyproject.toml

```diff
@@ -3,17 +3,14 @@ name = "detection_rules"
 version = "0.1.0"
 description = "Detection Rules is the home for rules used by Elastic Security. This repository is used for the development, maintenance, testing, validation, and release of rules for Elastic Security’s Detection Engine."
 readme = "README.md"
-requires-python = ">=3.8"
```

Comment:
If we want, we can update any other dependencies while we're here. but I only updated what was required.

Reply:
yea we should move to later versions on some of these, but would need to test